Source/Router.swift

```diff
@@ -205,51 +224,91 @@ public final class Router {
+     
+     - parameter path: The path used to match URL requests.
+     - parameter handler: A `RouteHandler` handler that will be used when the route is matched for a GET request
      */
-    public func post(relativePath: String, handler: RouteHandler) {
-        routes[relativePath] = (.POST, handler)
+    public func post(path: String, handler: RouteHandler) {
+        routes[path] = (.POST, handler)
     }
     
     /**
-     Registers a DEL request in a given relative path
+     Registers a DEL request with the given path
+     
+     The path is used togheter with the `Router.baseURL` to match requests. It can contain wildcard components prefixed by ":" that are later used to retreive the components of the request:
+     
+     - "/users/:userid" and "/users/1234" will produce [userid: 1234]
+     
+     Other than wildcards the components must be matched by the request.
+     The path should not contain paths that are already contained in the baseURL:
      
-     - parameter relativePath: A relative URL path to be registered
-     - parameter handler: A `RouteHandler` handler that will be used when intercepting the `path` with the `baseURL` for a DEL request
+     - base: "http://kakapo.com/api" -> path "/users/1234" ✅
+     - base: "http://kakapo.com/api" -> path "api/users/1234" ❌
+     
+     Trailing and leading slashes are not important for the route matching.
+     
+     - parameter path: The path used to match URL requests.
+     - parameter handler: A `RouteHandler` handler that will be used when the route is matched for a GET request
      */
-    public func del(relativePath: String, handler: RouteHandler) {
-        routes[relativePath] = (.DELETE, handler)
+    public func del(path: String, handler: RouteHandler) {
+        routes[path] = (.DELETE, handler)
     }
     
     /**
-     Registers a PUT request in a given relative path
+     Registers a PUT request with the given path
+     
+     The path is used togheter with the `Router.baseURL` to match requests. It can contain wildcard components prefixed by ":" that are later used to retreive the components of the request:
+     
+     - "/users/:userid" and "/users/1234" will produce [userid: 1234]
      
-     - parameter relativePath: A relative URL path to be registered
-     - parameter handler: A `RouteHandler` handler that will be used when intercepting the `path` with the `baseURL` for a PUT request
+     Other than wildcards the components must be matched by the request.
+     The path should not contain paths that are already contained in the baseURL:
+     
+     - base: "http://kakapo.com/api" -> path "/users/1234" ✅
+     - base: "http://kakapo.com/api" -> path "api/users/1234" ❌
+     
+     Trailing and leading slashes are not important for the route matching.
+     
+     - parameter path: The path used to match URL requests.
+     - parameter handler: A `RouteHandler` handler that will be used when the route is matched for a GET request
      */
```

Comment:
IMO no, because people don't search the documentation in a the files / README. The other documentation is mostly for us because is in private files. The touter is public and has to be documented properly (I missed that before and I didn't know how to use it properly) so you can just command click on a method/class to know how to use it.
What do you think?